lib/internal/fs/cp/cp-sync.js

```diff
@@ -145,24 +153,12 @@ function checkParentPathsSync(src, srcStat, dest) {
   return checkParentPathsSync(src, srcStat, destParent);
 }
 
-function handleFilterAndCopy(destStat, src, dest, opts) {
```

Comment:
change the name to make it consistent with [cp-async](https://github.com/nodejs/node/blob/aab9f3246eab93bc33a8b770fe00f7aa6e9b19a7/lib/internal/fs/cp/cp.js#L139)